code/sorting/test_sorting.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+from sorting.sorting import Sorting
+
+FIRST_CASE = [0, 7, 4]
+SECOND_CASE = [0, 45, 1,  8, 2, 4]
+THIRD_CASE = [0, 4, '45', '1', 2, 8]
+FOURTH_CASE = [1, '45', 3, 5, 'wef']
+
```

Comment:
Can you please add tests with float/negative values

Reply:
done